dev/breeze/src/airflow_ci/freespace.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""freespace.py for clean environment before start CI"""
+
+import shlex
+import subprocess
+from typing import List
+
+import click
+from rich.console import Console
+
+console = Console(force_terminal=True, color_system="standard", width=180)
+
+option_verbose = click.option(
+    "--verbose",
+    envvar='VERBOSE',
+    is_flag=True,
+    help="Print verbose information about free space steps",
+)
+
+option_dry_run = click.option(
+    "--dry-run",
+    is_flag=True,
+    help="Just prints commands without executing them",
+)
+
+
+@click.command()
+@option_verbose
+@option_dry_run
+def main(verbose, dry_run):
+    run_command(["sudo", "swapoff", "-a"], verbose, dry_run)
+    run_command(["sudo", "rm", "-f", "/swapfile"], verbose, dry_run)
+    run_command(["sudo", "apt", "clean", "||", "true"], verbose, dry_run)
```

Comment:
```suggestion
    run_command(["sudo", "apt", "clean", verbose, dry_run)
```

One other comment - the `|| truye` was more than part of the command - it was actuallly the way how we could avoid the command to fail. This is what `command || true` does

Reply:
In our case equivalent to that is setting `check` parameter  to False (it basically says - do not fail if the return code of the command is <> 0.

https://docs.python.org/3/library/subprocess.html

> If check is true, and the process exits with a non-zero exit code, a CalledProcessError exception will be raised. Attributes of that exception hold the arguments, the exit code, and stdout and stderr if they were captured.